observability/pillars/dashboard.md

```diff
@@ -0,0 +1,58 @@
+# Dashboard and Visualization
+
+## Overview
+
+TLDR - Dashboard to show patterns. What you want to achieve here is to derive a pattern - user, usage, search or anything that you have defined as your KPI.
+
+## Best Practices
+
+1. Identify your goal or KPI.
```

Comment:
nit: I would suggest avoiding the word 'you' and using more of a passive text style. This is the style used is almost all the places in the playbook.
For example instead of "Identify your goals or KPI", I would use more of a "imperative" style by rewording to 
"Firstly, it is important to identity the Key Performance Indicators (KPI) of the engagement. Identifying goals or KPI helps in defining what needs to be achieved."